README.md

```diff
@@ -0,0 +1,7 @@
+# JupyterHub spawner for Dataproc
+
+This repository provides a JupyterHub spawner for Dataproc.
+
+<!-- The following tools leverage this spawner:
```

Comment:
Can we pull in the existing documentation instead of deleting it?

Reply:
Modified. I did not keep the whole thing as trying locally does not work for example.
Also added a few conditions in the spawner so if we do not specify cluster configs, a cluster can still be spawn (the config concept did not exist as is in the original code)